### PR TITLE
Implement edge-to-edge on branch starter

### PR DESCRIPTION
**Pull request by johnshea**

### CODE REVIEW

File: app/src/main/java/com/example/bluromatic/ui/BluromaticScreen.kt [hunk 1]
```diff
@@ -60,18 +69,31 @@ import com.example.bluromatic.ui.theme.BluromaticTheme
...
+            blurAmountOptions = blurViewModel.blurAmount,
+            applyBlur = blurViewModel::applyBlur,
+            cancelWork = {},
+            modifier = Modifier
+                .verticalScroll(rememberScrollState())
+                .padding(
+                    start = dimensionResource(R.dimen.padding_small),
+                    top = dimensionResource(R.dimen.padding_small),
+                    end = dimensionResource(R.dimen.padding_small),
+                )
```
Comment (jhansitavva) [inline app/src/main/java/com/example/bluromatic/ui/BluromaticScreen.kt]: Could we change this to 

`.padding(dimensionResource(R.dimen.padding_medium))`

Where padding_medium is 16dp

`    <dimen name="padding_medium">16.dp</dimen>
`

File: app/src/main/java/com/example/bluromatic/ui/BluromaticScreen.kt [hunk 2]
```diff
@@ -173,7 +201,14 @@ fun BluromaticScreenContentPreview() {
             blurUiState = BlurUiState.Default,
             blurAmountOptions = listOf(BlurAmount(R.string.blur_lv_1, 1)),
             {},
-            {}
+            {},
+            modifier = Modifier
+                .verticalScroll(rememberScrollState())
```
Comment (jhansitavva) [inline app/src/main/java/com/example/bluromatic/ui/BluromaticScreen.kt]: We can remove `verticalScroll` from preview

File: app/src/main/java/com/example/bluromatic/ui/BluromaticScreen.kt [hunk 3]
```diff
@@ -173,7 +201,14 @@ fun BluromaticScreenContentPreview() {
             blurUiState = BlurUiState.Default,
```
Comment (jhansitavva) [inline app/src/main/java/com/example/bluromatic/ui/BluromaticScreen.kt:207]: Move this method below the private method

`private fun showBlurredImage(context: Context, currentUri: String) {`

File: app/src/main/java/com/example/bluromatic/ui/BluromaticScreen.kt (hunk 1 above)
Comment (johnshea) [reply app/src/main/java/com/example/bluromatic/ui/BluromaticScreen.kt]: Agreed. Done.

File: app/src/main/java/com/example/bluromatic/ui/BluromaticScreen.kt (hunk 2 above)
Comment (johnshea) [reply app/src/main/java/com/example/bluromatic/ui/BluromaticScreen.kt]: Removed.

File: app/src/main/java/com/example/bluromatic/ui/BluromaticScreen.kt (hunk 3 above)
Comment (johnshea) [reply app/src/main/java/com/example/bluromatic/ui/BluromaticScreen.kt:207]: Moved.

**Review by jhansitavva — APPROVED** [review]

**Review by jhansitavva — APPROVED** [review]

**Pull request merged**